src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Mvc.Module/.template.config/dotnetcli.host.json

```diff
@@ -12,6 +12,6 @@
   },
   "usageExamples": [
     "--framework net8.0",
-    "--orchard-version 2.0.0"
+    "--orchard-version 1.8.3"
```

Comment:
These changes are not related to this PR, right?

Reply:
No. Sorry. That was not intentional! I've reverted those changes.